server/node_modules

```diff
@@ -0,0 +1 @@
+/npm/node_modules
```

Comment:
What's going on with this? If I keep this around, this would mean if I do `npm install` on my local machine, it'll install to `/npm/node_modules`, yeah? I'd still want to run things locally for debug purposes, so this doesn't seem ideal.